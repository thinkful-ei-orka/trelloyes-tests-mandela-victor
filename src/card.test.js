import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

it('renders', () => {
     const div = document.createElement('div');
     ReactDOM.render(<Card />, div);
     ReactDOM.unmountComponentAtNode(div);
})

it('renders', () => {
     const tree = renderer
     .create(<Card className='Card' name='card name'/>)
     .toJSON();
     expect(tree).toMatchSnapshot();
})