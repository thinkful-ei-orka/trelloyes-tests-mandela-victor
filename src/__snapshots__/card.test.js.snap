// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3
    className="Card-title"
  />
  <p
    className="Card-content"
  />
</div>
`;
